src/tutorials/migrating.md

```diff
@@ -0,0 +1,54 @@
+---
+search:
+    keywords: ['migrating', 'upload']
+---
+
+# Migrating to Platform.sh
+
+Moving an already-built site to Platform.sh is generally straightforward.  For the most part, the only part that will vary from one framework to another is the details of the Platform.sh configuration files.  See the **Featured Framework** section or our [project templates](https://github.com/platformsh/) for more project-specific documentation.
```

Comment:
Hm. Our options here are:

1) No link (stet)
2) Link just to the app-containers page
3) Link different words to different doc pages
4) Put links to all 3 pages in a parenthetical

I don't like option 3 as it has too little context for the user.  I could deal with option 2 but it ignores the other two configuration files.  I'm not a huge fan of option as it feels very forced.  And since the config section is now very easy to find I'm OK with option 1.

Other thoughts?